common/lib/xmodule/xmodule/fields.py

```diff
@@ -299,3 +302,24 @@ def from_json(self, value):
         return Score(raw_earned, raw_possible)
 
     enforce_type = from_json
+
+
+class List(JSONField):
+    """
+    A field class for representing a list.
+
+    The value, as loaded or enforced, can either be None or a list.
+    """
+    _default = []
+
+    def from_json(self, value):
+        # Empty strings sometimes come in from OLX
+        if isinstance(value, basestring) and not value:
+            value = self._default
```

Comment:
This is a temporary hack to deal with some OLX with `html5_sources=""`.  I can fix this on `XBlock.fields.List`, unless there's another way to sanitize this value before its stored on the XBlock?

Reply:
@bradenmacdonald  Yep, cool, I've submitted https://github.com/edx/XBlock/pull/419 and dropped this change in favour of using that branch in b55a82ee6e9047445ef8b19f9cab93d47210d478.